src/pages/ProbeHybridisationQC.tsx

```diff
@@ -392,7 +392,6 @@ export default function ProbeHybridisationQC({ comments }: SectionComments) {
               <OperationCompleteModal
                 show={submissionResult !== undefined}
                 message="Probe Hybridisation QC recorded for all labware(s)"
-                onReset={reload}
```

Comment:
Why is the `onReset={reload}` removed here and several other places?

Reply:
OperationCompleteModal class is now changed with `react-router-dom` update. Now, the reset button within '`OperationCompleteModal`' calls '`reload`' function by default  and this `onReset` prop is required only if we want to override the default action.